Sources/MapboxCoreNavigation/NativeHandlersFactory.swift

```diff
@@ -1,6 +1,8 @@
 import MapboxNavigationNative
 import MapboxDirections
+import Foundation
 
+private let customConfigKey = "com.mapbox.navigation.custom-config"
```

Comment:
Good idea to prefix the key with a reverse DNS prefix. We should do the same for the `features` key that the `configHandle` getter expects.

Reply:
Would it be better to make this constant public but undocumented? It would reduce the potential for outdated code across the stack if the key ever changes.